src/containers/Layout/styles.sass

```diff
@@ -0,0 +1,10 @@
+.map-container
```

Comment:
also, this suggestion could be neglected if `react-toolbox` setup emphasizes on SASS syntax